src/components/UploadSection.svelte

```diff
@@ -84,6 +87,28 @@
                 uploadLoop.push(
                     pushMediaFilesToS3(postResponse[i].UploadUrl, files[i])
                 );
+                let exifData = null;
+
+                try {
+                    let exifData = await exifr.parse(files[i]);
+                } catch (e) {
+                    console.error(e);
+                }
+                let takenAtSec = null;
+
+                if (exifData) {
+                    takenAtSec = exifData.DateTimeOriginal.valueOf();
+                }
+
+                let pickedFile: UploadRequestResponse = {
+                    ObjectId: postResponse[i].ObjectId,
+                    ObjectKey: postResponse[i].ObjectKey,
+                    StatusName: "Pending upload",
+                    SizeBytes: files[i].size,
+                    TakenAtSec: takenAtSec,
+                };
+
+                uploadedFiles.push(pickedFile);
```

Comment:
![image](https://github.com/S1ckret-Labs/family-archive-spa/assets/52936598/6c35e258-866e-49d0-8df7-9715faece0ac)
after we pick file(the last ine what pending upload)
![image](https://github.com/S1ckret-Labs/family-archive-spa/assets/52936598/874ce6a6-60ba-4ec9-93f1-82ed581dedfb)
and after 1 minute when file is uploaded